packages/partykit/src/server.ts

```
@@ -78,6 +78,11 @@ export type Connection = WebSocket & {
   // It's also set as readonly, so we can't set it ourselves.
   // Instead, we'll use the `uri` property.
   uri: string;
+
+  // Underlying cloudflare types specify the type as any, but unknown is safer
+  // as it forces a cast, so let's add a optional generic overload
+  serializeAttachment<T = unknown>(attachment: T): void;
```

This is an unrelated change to improve type safety of these APIs. The original issue was unnecessarily hard to debug, because the `any` type swallowed what should have been an error on null property access.

The generic type here is optional, similar to how `storage.get` already works.